deploy/chart/templates/_packageserver.clusterserviceversion.yaml

```diff
@@ -48,6 +48,7 @@ spec:
           - ""
           resources:
           - configmaps
+          - namespaces
```

Comment:
The `package-server` needs RBAC to list namespaces in order to determine if the requested namespace exists (if it doesn't, return empty list like native Kubernetes resources).